js/g-cal.js

```diff
@@ -0,0 +1,229 @@
+var CLIENT_ID = 'CLIENT_ID';
```

Comment:
Is this file copied from somewhere or original? If it's partially or wholly copied, we'll need to know what it's licensed under before we can use it.